doc/grobi.conf

```diff
@@ -20,8 +20,8 @@ rules:
     # grobi takes the list of all the
     # outputs xrandr returns and verifies that HDMI2 and HDMI3 are connected, and
     # DP2-2 is present (but may be disconnected).
-    outputs_connected: [HDMI2, HDMI3]
-    outputs_present: [DP2-2]
+    outputs_connected: [{port: HDMI2}, {port: HDMI3}]
```

Comment:
Hm, I don't like that this makes the config file so much more complicated. What about if we just accept a string that can be either an output name or a (part of a) serial/manufacturer?

Reply:
Yeah, that was my thought as well. I do not know if someone might differentiate on the connected port, say use one config when the external display is connected via HDMI and another config, when it is connected via VGA for example. Then combining the port and the id into one string like `HDMI*;SAM-2618-808661557` might be a compromise. (Using some separator)
That being said, I currently do not have the need for this, so I would also be fine with using one string that can be either a output name or a monitor id.
When the config change comes, this could be revisited, I guess :wink:

Reply:
What about building a combined string of everything (`PORT-MANUF-MODEL-SERIAL`) and using `glob.Match()` to match against that? So the string would be `HDMI2-SAM-2618-808661557` and the pattern `HDMI*-SAM-2618` or so.

Do what you think is best, a string that can be both would be fine.